config_supported.go

```diff
@@ -2,6 +2,7 @@
 
 package wazero
 
+// CompilerSupported returns whether the compiler is supported in this environment.
 const CompilerSupported = true
```

Comment:
We accidentally exported this, so I moved it internal. Thanks for noticing as if you didn't we would have gone 1.0 with this exported without cause! https://github.com/tetratelabs/wazero/pull/597